specification/fist/resource-manager/Microsoft.IoTFirmwareDefense/preview/2023-02-08-preview/firmwares.json

```diff
@@ -0,0 +1,1983 @@
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Component"
+          },
+          "description": "The list of components.",
+          "x-ms-identifiers": [
+            "componentId"
+          ]
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The uri to fetch the next page of asset."
+        }
+      }
+    },
+    "BinaryHardeningFeatures": {
+      "type": "object",
+      "properties": {
+        "nx": {
+          "description": "NX flag.",
+          "type": "string",
+          "enum": [
+            "True",
+            "False"
+          ],
+          "x-ms-enum": {
+            "name": "nxFlag",
+            "modelAsString": true
+          }
+        },
+        "pie": {
+          "description": "PIE flag.",
+          "type": "string",
+          "enum": [
+            "True",
+            "False"
+          ],
+          "x-ms-enum": {
+            "name": "pieFlag",
+            "modelAsString": true
+          }
+        },
+        "relro": {
+          "description": "RELRO flag.",
+          "type": "string",
+          "enum": [
+            "True",
+            "False"
+          ],
+          "x-ms-enum": {
+            "name": "relroFlag",
+            "modelAsString": true
+          }
+        },
+        "canary": {
+          "description": "Canary flag.",
+          "type": "string",
+          "enum": [
+            "True",
+            "False"
+          ],
+          "x-ms-enum": {
+            "name": "canaryFlag",
+            "modelAsString": true
+          }
+        },
+        "stripped": {
+          "description": "Stripped flag.",
+          "type": "string",
+          "enum": [
+            "True",
+            "False"
+          ],
+          "x-ms-enum": {
+            "name": "strippedFlag",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Binary hardening features."
+    },
+    "BinaryHardening": {
+      "type": "object",
+      "properties": {
+        "binaryHardeningId": {
+          "description": "ID for the binary hardening result.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "features": {
+          "title": "BinaryHardeningFeatures",
+          "description": "Binary hardening features.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/BinaryHardeningFeatures"
+        },
+        "architecture": {
+          "description": "The architecture of the uploaded firmware.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "path": {
+          "description": "path for binary hardening.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "class": {
+          "description": "class for binary hardening.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "runpath": {
```

Comment:
>runpath [](http://example.com/codeflow?start=9&length=7)

I'd suggest naming this 'runPath' in camel case, instead. Then it won't look misspelled and we won't need to ad it to custom-words.

Reply:
same as above, it's known to be spelled like this for linux binary compilation parameters. If it's an issue we could change it to camel case but would prefer not to

Reply:
You can go with cSpell in that case.